vizro-core/examples/_dev/app.py

```diff
@@ -3,77 +3,29 @@
 import vizro.models as vm
 import vizro.plotly.express as px
 from vizro import Vizro
+from vizro.models._components.form._text_area import TextArea
+from vizro.models._components.form._user_input import UserInput
 
-gapminder = px.data.gapminder()
+iris = px.data.iris()
```

Comment:
Do we need to undo the changes for this file before merging on the main? 

It seems that we've changed the file content in [this PR](https://github.com/mckinsey/vizro/pull/277). I'm wondering what is the final content for this example that we should keep.